segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/ZooKeeperServiceRunner.java

```diff
@@ -69,21 +108,20 @@ public void start() throws Exception {
             s.shutdown();
             throw new IllegalStateException("Already started.");
         }
-
-        this.serverFactory.set(new NIOServerCnxnFactory());
-        val address = LOOPBACK_ADDRESS.getHostAddress() + ":" + this.zkPort;
+        this.serverFactory.set(NettyServerCnxnFactory.createFactory());
+        val address = "localhost:" + this.zkPort;
         log.info("Starting Zookeeper server at " + address + " ...");
-        this.serverFactory.get().configure(new InetSocketAddress(LOOPBACK_ADDRESS, this.zkPort), 1000);
+        this.serverFactory.get().configure(new InetSocketAddress("localhost", this.zkPort), 1000, secureZK);
```

Comment:
Please add `LOOPBACK_ADDRESS` back and change its contents to "localhost". Then replace all string occurrences of "localhost" in this file with references to that private member

Reply:
Fixed.